### PR TITLE
fix to new project

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
this solves #1280

### CODE REVIEW

**Comment by JIMENOFONSECA** [comment]
@shizhongming please check if this works now for you.

File: cea/datamanagement/create_new_project.py [hunk 1]
```diff
@@ -26,37 +28,39 @@
 COLUMNS_ZONE_GEOMETRY = ['Name', 'floors_bg', 'floors_ag', 'height_bg', 'height_ag']
```
Comment (daren-thomas) [inline cea/datamanagement/create_new_project.py]: let's keep in mind that the end goal should be to have this defined somewhere centrally for the whole CEA... i think the glossary efforts could end up producing something like this, maybe a YAML file with information about input and output formats?

**Comment by shizhongming** [comment]
Hi @JIMENOFONSECA 
I was trying to test it, while an error occurred.
Missing the parameter "occupancy_types"
`C:\Users\Zhongming\Anaconda2\envs\cea\python.exe C:/Users/Zhongming/Documents/CityEnergyAnalyst/cea/datamanagement/create_new_project.py
Traceback (most recent call last):
  File "C:/Users/Zhongming/Documents/CityEnergyAnalyst/cea/datamanagement/create_new_project.py", line 141, in <module>
    main(cea.config.Configuration())
  File "C:/Users/Zhongming/Documents/CityEnergyAnalyst/cea/datamanagement/create_new_project.py", line 125, in main
    print("Running create-new-project with occupancy-types = %s" % config.create_new_project.occupancy_types)
  File "C:\Users\Zhongming\Documents\CityEnergyAnalyst\cea\config.py", line 193, in __getattr__
    raise AttributeError("Parameter not found: %s" % item)
AttributeError: Parameter not found: occupancy_types
Running create-new-project with project = WTP_03
Running create-new-project with scenario = test

Process finished with exit code 1
`

**Comment by JIMENOFONSECA** [comment]
@shizhongming this is ready to be tested again. would you mind?

**Comment by shizhongming** [comment]
@JIMENOFONSECA Thanks for the updates..the script runs now, but it is still replacing the existing occupancy.dbf and age.dbf with the default ones...

**Comment by shizhongming** [comment]
@JIMENOFONSECA I think all good. Ready to merge~ :)

**Pull request merged**